jquery.flot.composeImages.js

```diff
@@ -238,6 +254,7 @@
             copySVGToImg(srcCanvasOrSvg, destImg);
         }
 
+        destImg.srcImgTagName = srcCanvasOrSvg.tagName;
```

Comment:
Why?

Reply:
See your previous comment. That is where it is used.
destImg will become sources[i] in prepareImagesToBeComposed function.